src/Nethermind/Nethermind.Merge.Plugin.Test/EngineModuleTests.V3.cs

```diff
@@ -43,7 +43,7 @@ public async Task NewPayloadV1_should_decline_post_cancun()
 
         ResultWrapper<PayloadStatusV1> errorCode = (await rpcModule.engine_newPayloadV1(executionPayload));
 
-        Assert.That(errorCode.ErrorCode, Is.EqualTo(ErrorCodes.InvalidParams));
```

Comment:
I assume that we should return InvalidParams still here
So if you pass newPayloadV1 with post-cancun timestamp your validation should fail once checking that BlobVersionedHashes are null and it should give you InvalidParams
UnsupportedFork will be when you pass newPayloadV3 with pre-cancun timestamp

Reply:
The idea is to show this error if we have Cancun timestamp in V2 and pre-Cancun timestamp in V3